### DISCUSSION TITLE
Shortcut not working on HomePod Mini

### DISCUSSION THREAD

**Discussion by Melodicaxe1** [Yue-Yang/ChatGPT-Siri#99] [General]
Hi there,

Shortcut works perfect on iPhone and iPad however when running on HomePod Mini, I ask to run the shortcut but then get this error...

"Sorry I am still having problems with connection.."

Both iPad, iPhone and HomePod Mini are on the same wifi network.

Thanks,

**Comment by jjustinwilson** [comment]
I've had the same issue. I was looking forward to being able to talk with ChatGPT.

**Comment by Yue-Yang** [comment]
I dont know because I dont use homepod, but my user told me he can use both on homepod and carplay

https://twitter.com/gwthompson/status/1632452290148802561?s=20

**Comment by Melodicaxe1** [comment]
Has anyone else been able to get the shortcut to work on Homepod?
